### ISSUE TITLE
Modify so clicking on labels takes you to a link?

### ISSUE THREAD

**Issue by ChrisRahe** [ekallevig/jShowOff#63]
We are using jShowOff on our church site using custom labels at the bottom for each slide. A lot of people are confused and don't realize that when they click on the text labels it just loads the slide. They don't realize they then have to click on the image in order to go to the specified page. Is there a way to modify it so that clicking on the text label actually takes you to the same link as if you clicked on the corresponding image?

I'm pretty good with HTML and CSS but I'm very much an amateur when it comes to PHP and JavaScript - I can muddle through simple changes, but I can't always figure this stuff out by myself.

**Comment by john-benedict** [comment]
I have a client request for this feature too.

Anyone have a solution?

**Comment by john-benedict** [comment]
If you're using the production (non-minimized) version -- look for line 232:

$('<a class="jshowoff-slidelink-'+i+' '+uniqueClass+'-slidelink-'+i+'" href="#null">'+linktext+'</a>').bind('click', {index:i}, function(e){ goToAndPause(e.data.index); return false; }).appendTo('.'+uniqueClass+'-slidelinks');

Replace with:

$('<a class="jshowoff-slidelink-'+i+' '+uniqueClass+'-slidelink-'+i+'" href="#null">'+linktext+'</a>').bind('mouseenter', {index:i}, function(e){ goToAndPause(e.data.index); return false; }).appendTo('.'+uniqueClass+'-slidelinks');

Basically -- just replace click with mouseenter after bind.

**Comment by ChrisRahe** [comment]
Thanks for responding. I’m guessing I have the minimized version because I’m not finding that bit of code anywhere in the js file.

---

Chris Rahe, web services director, Mennonite Church USA

**Comment by john-benedict** [comment]
Yeah -- you have to get the working version.

You can grab it here:

https://github.com/ekallevig/jShowOff
